### ISSUE TITLE
Can't parse 'center'. Sequence item with index 0 has a wrong type

### ISSUE THREAD

**Issue by hxxgogogo** [Poissonfish/GRID#5]
Describe the bug
Hi, I have successfully downloaded the photo grid and successfully opened it, but when I choose the demo and then click next, the program flashed back. I don't know why.

Screenshots

![图片](https://user-images.githubusercontent.com/85552105/121452562-3980f580-c9d2-11eb-9805-34c994863866.png)

I click next and then flashed back.

![图片](https://user-images.githubusercontent.com/85552105/121452582-44d42100-c9d2-11eb-82e3-6d43d931cd75.png)

Desktop (please complete the following information):
 - OS: Win10
 - Browser: FireFox
 - Version :1.2.15

Additional context: I am interested in this program , but I don't know what should I do now. I would appreciate it if you could help me

**Comment by Poissonfish** [comment]
Hi,

You might want to define AOI correctly in the previous step. The details can be found here: https://poissonfish.github.io/GRID/ch2_interface/aoi.html

**Comment by hxxgogogo** [comment]
Thank you, James. I have defined my interesed AOI area in the previous step, but it still flashed down. So I changed parameters to change it, but it still not work.It seems that one of the variables has a null value.

![图片](https://user-images.githubusercontent.com/85552105/121980943-6c0a6400-cdbf-11eb-8760-911ae6b6408b.png)

Then I click next, it still flashed down.

![图片](https://user-images.githubusercontent.com/85552105/121980733-12a23500-cdbf-11eb-98c6-82864d623f1f.png)
